linux_os/guide/services/xwindows/disabling_xwindows/package_xorg-x11-server-common_removed/rule.yml

```diff
@@ -47,7 +47,18 @@ ocil: |-
     The output should be:
     <pre>package xorg-x11-server-common is not installed</pre>
 
+warnings:
+    - general: |-
+        If you use a system with Graphical User Interface (GUI), by removing the package xorg-x11-server-common, it can
+        potentially remove the graphical target and it might bring your system to an inconsistent state where
+        additional configuration can be required to access the system again. Only remove this package if you are sure.
+
 template:
     name: package_removed
     vars:
         pkgname: xorg-x11-server-common
+    backends:
```

Comment:
Remediations should be kept. Tailoring should be used instead.

Reply:
> Sounds like the installer has a bug or should otherwise notify the user about package conflicts.

This is acknowledged in the PR description. 
I see this as a mitigation for the technical problem, which is the machine seems broken after installation.

Another possible approach is to remove this rule from "templation" and extend the bash remediation to also change the target to `multi-user`.

To solve the conflict of expectations (with or without GUI) a change in OAA is needed anyway.

Reply:
> Another possible approach is to remove this rule from "templation" and extend the bash remediation to also change the target to `multi-user`.

Difference between the approaches being that in one case the GUI is there (the intended server install config wins), in the other the GUI is not there (the intended policy wins).

Reply:
I have reverted back the remediations and took a different approach. I added checks and remediation for rule [xwindows_runlevel_target](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/xwindows/disabling_xwindows/xwindows_runlevel_target/rule.yml) which will select the correct target for the system and it will prevent from ending up with a black screen. This rule has been selected in all profiles (but the one from RHEL6 because it is not applicable) that select the rule: `package_xorg-x11-server-common_removed`.